src/lib/customRequest/customRequest.test.ts

```diff
@@ -0,0 +1,54 @@
+import { Client } from '@microsoft/microsoft-graph-client'
```

Comment:
I realize not all of our projects adhere, but I think most use `.spec` for unit tests. Not a huge deal, but I'd say let's go with the majority on new stuff.